imageProcessing/at-u2-v0.cpp

```diff
@@ -367,18 +384,23 @@ int main( int argc, char* argv[] )
           const Calculus::PrimalForm2 mg1 = functions::dec::diagonal( m ) * AT.getG( 1 );
           const Calculus::PrimalForm2 mg2 = functions::dec::diagonal( m ) * AT.getG( 2 );
           functions::dec::threeForms2ToRGBColorImage
-            ( AT.calculus, mg0, mg1, mg2, image_mg, 0.0, 1.0, 1 ); 
+            ( AT.calculus, mg0, mg1, mg2, image_mg, 0.0, 1.0, 1 );
           PPMWriter<ColorImage, functors::Identity >::exportPPM( ossGM.str(), image_mg );
         }
     }
-  else 
+  else
     AT.setAlpha( a );
-  
+
+  ofstream energieFile;
+  energieFile.open("./imageProcessing/Resultat/SansMetrique_calcul_energie.txt", ofstream::out | ofstream::app);
```

Comment:
This file is not part of this PR, and also are you sure you want to load it like this?